RealEstate/0 database.sql

```diff
@@ -0,0 +1 @@
+create database RealEstate
```

Comment:
Please add to the database file the following:
make sure you are in the master database
please drop the database before you create it

(BTW we should probably not put in the file to drop DB if exists-this way if you run it by accident it will not run.?..?..)